app/components/pipeline-options/component.js

```diff
@@ -216,6 +216,31 @@ export default Component.extend({
       } finally {
         this.set('isUpdatingMetricsDowntimeJobs', false);
       }
+    },
+
+    async updateShowPRJobs(showPRJobs) {
+      const pipelineId = this.get('pipeline.id');
+
+      let pipelinePreference = await this.store.queryRecord('preference/pipeline', {
+        filter: { pipelineId }
+      });
```

Comment:
Should you just use the `getUserPreference` method here?
```suggestion
      let pipelinePreference = await this.shuttle.getUserPreference(pipelineId);
```

Reply:
`getUserPreference` will trigger an API call to compare local and remove settings.

```const remotePreferences = await this.getUserSetting();```